package.json

```diff
@@ -58,21 +59,10 @@
     "typescript": "^4.3.5"
   },
   "peerDependencies": {
-    "@ethersproject/hardware-wallets": "^5.0.14",
+    "ethers": "^5.5.0",
```

Comment:
Why is do we need to add the whole ethers.js lib as peer deps

the `@ethersproject/hardware-wallets`  is actually optional dependency for use that want to use ledger

Would like to keep hardhat-deploy agnostic otherwise and if possible not require users to install ethers

Reply:
good